### PR TITLE
fix: pscanrulesAlpha: Improper CHANGELOG note placement

### PR DESCRIPTION
**Pull request by kingthorin**
Per: https://github.com/zaproxy/zap-extensions/pull/3306#discussion_r745693123

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

**Comment by thc202** [comment]
There's a typo in the add-on ID.

File: addOns/pscanrulesAlpha/CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Change
```
Comment (thc202) [inline addOns/pscanrulesAlpha/CHANGELOG.md:7]: `Changed`

**Comment by kingthorin** [comment]
DOH, also done :wink:

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**